### DISCUSSION TITLE
Problem icon hidden  or process admin in windows bar

### DISCUSSION THREAD

**Discussion by LeanPC** [LizardByte/community#204] [Sunshine] [answered]
Hi Guys,
when sunshine starts automatically in the hidden icons section in the windows bar the process is shown multiplied and sometimes the process does not appear in the task manager.

version 0.19.1

![Captura de pantalla 2023-03-31 155341](https://user-images.githubusercontent.com/17548540/229205765-c49c9891-0e51-4a01-b6a7-41a952f8cbaf.png)

**Comment by ReenigneArcher** [comment, marked as answer]
If you have multiple icons, then either Sunshine has crashed or you're running multiple instances.
